core/src/test/scala/integration/kafka/api/QuotasTest.scala

```diff
@@ -170,22 +194,21 @@ class QuotasTest extends KafkaServerTestHarness {
     Assert.assertEquals("Should not have been throttled", Double.NaN, allMetrics(consumerMetricName).value())
   }
 
-  def produce(p: KafkaProducer[Array[Byte], Array[Byte]], count: Int): Int = {
-    var numBytesProduced = 0
+  def produce(p: KafkaProducer[Array[Byte], Array[Byte]], count: Int) = {
     for (i <- 0 to count) {
-      val payload = i.toString.getBytes
-      numBytesProduced += payload.length
+      val payload = Array[Byte](100)
       p.send(new ProducerRecord[Array[Byte], Array[Byte]](topic1, null, null, payload),
              new ErrorLoggingCallback(topic1, null, null, true)).get()
       Thread.sleep(1)
     }
-    numBytesProduced
   }
 
+
   def consume(consumer: KafkaConsumer[Array[Byte], Array[Byte]], numRecords: Int) {
     consumer.subscribe(topic1)
     var numConsumed = 0
     while (numConsumed < numRecords) {
+      import scala.collection.JavaConversions._
```

Comment:
It's needed because of the `poll` below (and the for comprehension). Having said that, I'd use `poll(100).asJava` instead. More concise and less magic.